storage.go

```diff
@@ -234,26 +237,26 @@ func (s *Storage) createMultipart(ctx context.Context, path string, opt pairStor
 	o.ID = rp
 	o.Path = path
 	o.Mode |= ModePart
-	o.SetMultipartID(aws.StringValue(output.UploadId))
+	o.SetMultipartID(*aws.String(*output.UploadId))
```

Comment:
Maybe you want to use https://github.com/aws/aws-sdk-go-v2/blob/main/aws/from_ptr.go

Reply:
Oh,that's more elagant. It's wired to use many * on one field. 
I will fixed it soon.

Reply:
Yes, it's much better than using two *.I will fixed it right now.